editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -2242,54 +2246,61 @@ void TileSetAtlasSourceEditor::edit(Ref<TileSet> p_tile_set, TileSetAtlasSource
 	_update_current_tile_data_editor();
 }
 
-void TileSetAtlasSourceEditor::init_source() {
+void TileSetAtlasSourceEditor::init_source(const Vector<Ref<TileSetAtlasSource>> &p_atlases) {
```

Comment:
Since we have several sources now:
```suggestion
void TileSetAtlasSourceEditor::init_sources(const Vector<Ref<TileSetAtlasSource>> &p_atlases) {
```

Reply:
I could change to `init_atlases` or `init_new_atlases`.